### PR TITLE
Bump RustCrypto dependencies

### PR DESCRIPTION
**Pull request by tarcieri**
- `aes` v0.8
- `ctr` v0.9
- `k256` v0.11
- `scrypt` v0.10

### CODE REVIEW

**Comment by tarcieri** [comment]
I can bump `k256` as well if you'd like

**Comment by roynalnaruto** [comment]
@tarcieri Thank you :)

Yes may be you could also bump `k255` in this PR? I'll release a new version of this crate upon merging!

**Comment by tarcieri** [comment]
Bumped `k256` and fixed an unrelated clippy nit

File: src/lib.rs [hunk 1]
```diff
@@ -144,20 +146,18 @@ where
...
+    let mut pk = keystore.crypto.ciphertext;
     decryptor.apply_keystream(&mut pk);
 
     Ok(pk)
 }
 
 /// Encrypts the given private key using the [Scrypt](https://tools.ietf.org/html/rfc7914.html)
 /// password-based key derivation function, and stores it in the provided directory. On success, it
-/// returns the `id` (Uuid) generated for this keystore.
+/// retur ns the `id` (Uuid) generated for this keystore.
```
Comment (roynalnaruto) [inline src/lib.rs]: I think this is a typo?

**Review by roynalnaruto — APPROVED** [review]
Just a nitpick. I have approved and will merge once you confirm/edit the typo I pointed out

File: src/lib.rs (hunk 1 above)
Comment (tarcieri) [reply src/lib.rs]: Oops, not sure how that happened. Fixed.

**Comment by tarcieri** [comment]
Fixed. Went ahead and squashed the commits too.

**Pull request merged**